tools/test.py

```diff
@@ -174,6 +174,21 @@ def main():
                               **cfg.data.get('test_dataloader', {}))
     data_loader = build_dataloader(dataset, **dataloader_setting)
 
+    if cfg.model.get('backbone', None):
+        # remove pretrain steps for testing
+        if hasattr(cfg.model.backbone, 'pretrained'):
+            cfg.model.backbone.pretrained = None
+        if hasattr(cfg.model.backbone, 'pretrained2d'):
+            cfg.model.backbone.pretrained2d = False
+
+        # For slowfast case
+        if cfg.model.backbone.get('slow_pathway', None):
```

Comment:
We can recursively find all `pretrained` in the model config. There's no need to write a specific case for SlowFast